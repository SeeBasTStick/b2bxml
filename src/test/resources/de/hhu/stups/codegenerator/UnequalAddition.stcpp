int main() {
    Unequal comparison;
    comparison.calculate();
    cout << comparison.getRes();
    return 0;
}