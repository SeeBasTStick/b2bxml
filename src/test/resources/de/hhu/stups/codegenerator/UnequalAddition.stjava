public static void main(String[] args) {
    Unequal comparison = new Unequal();
    comparison.calculate();
    System.out.println(comparison.getRes());
}