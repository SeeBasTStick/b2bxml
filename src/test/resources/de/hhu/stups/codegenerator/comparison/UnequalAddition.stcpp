int main() {
    Unequal comparison;
    comparison.simulate();
    cout << comparison.getCounter();
    return 0;
}