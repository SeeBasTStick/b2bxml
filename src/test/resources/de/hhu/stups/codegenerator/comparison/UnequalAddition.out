public static void main(String[] args) {
    Unequal comparison = new Unequal();
    comparison.simulate();
    System.out.println(comparison.counter);
}